Armstrong_Number/Armstrong_Number.rb

```diff
@@ -0,0 +1,44 @@
+# A positive integer of n digits is called an Armstrong number of order n (order is number of digits) if
+# abcd... = pow(a,n) + pow(b,n) + pow(c,n) + pow(d,n) + ....
+
+# To calculate the order of the number
+def order(num)
+  # variable to store of the number
```

Comment:
Well checking is pretty basic. Like this line is inside the function block, so there should be 4 spaces before the start of comment, and it is 2 here.

Reply:
just wait one minute

Reply:
resolved.